uroboros.py

```diff
@@ -97,7 +97,7 @@ def create_uroboros(graph, mode, max_iter, freqs):
         size = len(graph)
         freqs = get_freqs(graph, mode)
         skip = collect_skip(graph, freqs, sort_by)
-        for word, _ in sorted(freqs.iteritems(), key=sort_by):
+        for word, _ in sorted(freqs.iteritems(), key=lambda x: (sort_by, x[0])):
```

Comment:
it is just an extra sorting providing that words of the same frequency come always in the same order. i should have written it in the get_sort_mode() as a mode called "rare_plus_alpha" 
is it ok that way?